code/src/components/AddTask.js

```diff
@@ -0,0 +1,87 @@
+import styled from 'styled-components';
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import todos from 'reducers/todos';
+import uniqid from 'uniqid';
+
+const AddTask = () => {
+  const dispatch = useDispatch()
+  const [input, setInput] = useState('');
+
+  // Adds new task to TaskList when submitting form
+  const submitHandler = (e) => {
+    e.preventDefault()
+    dispatch(todos.actions.addTask({ // from reducer
+      id: uniqid(),
+      text: input,
+      isDone: false
+    }))
```

Comment:
perhaps I would have placed this segment in the todo.js reducer- addTask. To keep everything together and the component more "tidy".